i18n/rpi-imager_tr.ts

```diff
@@ -0,0 +1,414 @@
+        <location filename="../driveformatthread.cpp" line="124"/>
+        <location filename="../driveformatthread.cpp" line="185"/>
+        <source>Error partitioning: %1</source>
+        <translation>Bölümleme hatası: %1</translation>
+    </message>
+    <message>
+        <location filename="../driveformatthread.cpp" line="84"/>
+        <source>Error starting fat32format</source>
+        <translation>fat32format başlatılırken hata oluştu</translation>
+    </message>
+    <message>
+        <location filename="../driveformatthread.cpp" line="94"/>
+        <source>Error running fat32format: %1</source>
+        <translation>fat32format çalıştırılırken hata oluştu: %1</translation>
+    </message>
+    <message>
+        <location filename="../driveformatthread.cpp" line="104"/>
+        <source>Error determining new drive letter</source>
+        <translation>Yeni sürücü harfi belirlenirken hata oluştu</translation>
+    </message>
+    <message>
+        <location filename="../driveformatthread.cpp" line="109"/>
+        <source>Invalid device: %1</source>
+        <translation>Geçersiz cihaz: %1</translation>
+    </message>
+    <message>
+        <location filename="../driveformatthread.cpp" line="146"/>
+        <source>Error formatting (through udisks2)</source>
+        <translation>Hatalı biçimlendirme (udisks2 aracılığıyla)</translation>
+    </message>
+    <message>
+        <location filename="../driveformatthread.cpp" line="174"/>
+        <source>Error starting sfdisk</source>
+        <translation>sfdisk başlatılırken hata oluştu</translation>
+    </message>
+    <message>
+        <location filename="../driveformatthread.cpp" line="196"/>
+        <source>Error starting mkfs.fat</source>
+        <translation>mkfs.fat başlatılırken hata oluştu</translation>
+    </message>
+    <message>
+        <location filename="../driveformatthread.cpp" line="206"/>
+        <source>Error running mkfs.fat: %1</source>
+        <translation>mkfs.fat çalıştırılırken hata oluştu: %1</translation>
+    </message>
+    <message>
+        <location filename="../driveformatthread.cpp" line="213"/>
+        <source>Formatting not implemented for this platform</source>
+        <translation>Bu platform için biçimlendirme uygulanmadı</translation>
+    </message>
+</context>
+<context>
+    <name>ImageWriter</name>
+    <message>
+        <location filename="../imagewriter.cpp" line="170"/>
+        <source>Storage capacity is not large enough.
+Needs to be at least %1 GB</source>
+        <translation>Depolama kapasitesi yeterince büyük değil. 
+En az %1 GB olması gerekiyor</translation>
+    </message>
+    <message>
+        <location filename="../imagewriter.cpp" line="176"/>
+        <source>Input file is not a valid disk image.
+File size %1 bytes is not a multiple of 512 bytes.</source>
+        <translation type="unfinished">Giriş dosyası geçerli bir disk görüntüsü değil. 
```

Comment:
This one still says "unfinished" ?

Reply:
I put it for control. Translation is really importand. I checked Turkish Dictionary to compare my translations whether it is true